### PR TITLE
K8s: Release viking 7.8

### PR DESCRIPTION
**Pull request by kaitlynmichael**
DOC-4409
DOC-4152

### CODE REVIEW

**Comment by paoloredis** [comment]
The versioning picker expects `content/operate/kubernetes/_index.md` to have:

```
linkTitle: Redis for Kubernetes
```

File: layouts/partials/docs-nav.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
         </button>
         <div id="versionDropdownKubernetes" class="menu__version-selector__list version-selector-control">
             <a href="https://redis.io/docs/latest/operate/kubernetes/" id="kubernetes-version-select-latest" onclick="_setSelectedVersion('kubernetes', 'latest')">latest</a>
+            <a href="https://dev.learn.redis.com/docs/latest/operate/kubernetes/7.4.6/" id="kubernetes-version-select-7.4.6" onclick="_setSelectedVersion('kubernetes', 'v7.4.6')">v7.4.6</a>
```
Comment (paoloredis) [inline layouts/partials/docs-nav.html:29]: My bad, the href link should actually be:

https://redis.io/docs/latest/operate/kubernetes/7.4.6/

File: content/operate/kubernetes/upgrade/upgrade-redis-cluster.md [hunk 2]
```diff
@@ -19,13 +19,43 @@ Redis implements rolling updates for software upgrades in Kubernetes deployments
...
+
+Your Redis Enterprise clusters must be running version 7.4.2-2 or later before upgrading to 7.8.2-2. See the [7.4 upgrade]({{<relref "/operate/kubernetes/7.4.6/upgrade">}}) for detailed steps.
+
+### Redis database version
+
+Your Redis databases must be running version 7.2 or later before upgrading your cluster version to 7.8.2-2. See [upgrade databases](#upgrade-databases) for detailed steps. You can find your database version in the [REDB `spec.redisVersion` field]({{<relref "/operate/kubernetes/reference/redis_enterprise_database_api#redisversion" >}}).
+
+### RHEL9-compatible modules
+
+Upgrading to Redis operator version 7.8.2-2 requires migrating your Redis Enterprise nodes to RHEL9 from either Ubuntu 18 or RHEL8. If your databases use modules, you need to manually install modules compatibile with RHEL9.
```
Comment (heinrich-redislabs) [inline content/operate/kubernetes/upgrade/upgrade-redis-cluster.md:40]: the word `requires` may be misleading here. It's not like users are required to take a special action.
Simply upgrading the operator and the Redis Enterprise cluster results in migration from RHEL8/Ubuntu18 to RHEL9.

**Review by heinrich-redislabs — APPROVED** [review]
I went over the document and left one non-blocking comment.

File: content/operate/kubernetes/upgrade/upgrade-redis-cluster.md (hunk 2 above)
Comment (kaitlynmichael) [reply content/operate/kubernetes/upgrade/upgrade-redis-cluster.md:40]: I'll change this to "involves"

**Review by cmilesb — APPROVED** [review]
Looks good to me!

**Comment by kaitlynmichael** [comment]
Approved by @bascar via slack

**Review by bascar — COMMENTED** [review]
LGTM!

**Pull request merged**